base_layer/core/src/transactions/aggregated_body.rs

```diff
@@ -250,22 +250,18 @@ impl AggregateBody {
                     warn!(target: LOG_TARGET, "Coinbase {} found with maturity set too low", utxo);
                     return Err(TransactionError::InvalidCoinbaseMaturity);
                 }
-                coinbase_utxo = Some(utxo);
+                coinbase_utxo_sum = &coinbase_utxo_sum + &utxo.commitment;
```

Comment:
Lets add a unit test with say 50 coinbases in one block

Reply:
Existing tests are fine, we don't need to ber extravagant